internal/storage/environments/environments.go

```diff
@@ -281,6 +283,38 @@ func (f *EnvironmentFactory) Create(ctx context.Context, name string, envConf *c
 			if err != nil {
 				return nil, fmt.Errorf("failed to setup gitlab scm: %w", err)
 			}
+
+		case config.AzureSCMType:
+			repoURL, err := azureparserv1.NewAzureParserWithURL(repo.GetRemote())
```

Comment:
shouldnt our existing git url parsing handle this? https://github.com/flipt-io/flipt/blob/33733d13662d190a312726c1e89fbced9ad6370e/internal/coss/storage/environments/git/url.go#L18

it should handle azure by the looks of it https://github.com/kubescape/go-git-url/blob/d27eb58fdc8cd9154b6fdef2c355cdfe9c3dfde1/init.go#L24

Reply:
`gitURL` is `IGitURL` and that interface doesn't has `Project` function.